invenio_app_ils/patrons/views.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio App ILS Patrons views."""
+
+from flask import Blueprint
+from flask import abort, current_app
+from flask_login import current_user
+from invenio_circulation.search.api import search_by_patron_item_or_document
```

Comment:
here you might want to sort correctly your import: `isort`, that is executed in the script `run-style-checks.sh` will fail.
Actually it does not because it is bugged :)

Can you please change the file `run-style-check.sh` and put:
```
isort invenio_app_ils tests --check-only --diff
```

For code formatting, we use [black](https://github.com/psf/black) with max line length to `-l 79`. You can eventually install it and execute to format your code.
I have setup both `isort` and `black` in my PyCharm in case you need.